.babelrc

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "development": {
+      "presets": ["next/babel"],
+      "plugins": [["styled-components", { "ssr": true }]]
+
+    },
+    "production": {
+      "presets": ["next/babel"],
+      "plugins": [["styled-components", { "ssr": true }]]
+
+    },
+    "test": {
+      "presets": ["next/babel","@babel/preset-typescript"],
+      "plugins": ["styled-jsx/babel-test"]
+    }
+  }
+}
```

Comment:
we don't need this for styled-components after recent nextjs update, we can continue using swc compiler, it's 17x faster than babel.